internal/grpc/token/v1/token.pb.go

```diff
@@ -3,8 +3,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.35.1
-// 	protoc        v5.28.2
+// 	protoc-gen-go v1.34.2
```

Comment:
Please use latest version of `protoc-gen-go` & `protoc`    
or    
`protoc-gen-go v1.35.1`
`protoc        v5.28.2`